src/sql/workbench/parts/connection/common/serverInfoContextKey.ts

```diff
@@ -5,18 +5,21 @@
 
 import { RawContextKey, IContextKeyService, IContextKey } from 'vs/platform/contextkey/common/contextkey';
 import { ServerInfo } from 'azdata';
+import { DatabaseEngineEdition } from 'sql/workbench/api/common/sqlExtHostTypes';
 
 export class ServerInfoContextKey implements IContextKey<ServerInfo> {
 
 	static ServerInfo = new RawContextKey<ServerInfo>('serverInfo', undefined);
 	static ServerMajorVersion = new RawContextKey<string>('serverMajorVersion', undefined);
 	static IsCloud = new RawContextKey<boolean>('isCloud', undefined);
 	static IsBigDataCluster = new RawContextKey<boolean>('isBigDataCluster', undefined);
+	static IsSqlOnDemand = new RawContextKey<boolean>('isSqlOnDemand', undefined);
 
```

Comment:
I'd prefer this to be EngineEdition - same as the other places - and then just check for 11 wherever this is used